raster-foundry/blasterjs
# Feature/sml/theme

## Target Branch
Note this is a request to merge to `feature/theme`, not `develop`. All theme-related PRs should target `feature/theme` until the theme upgrade is complete.

## Overview

Updates the following components to the new theme system:

- Card
- Text
- Heading
- A
- Divider
- Image
- ProgressBar
- Swatch

Docz pages have been updated for accuracy, but not overhauled in any way.


### Checklist


### Upgrade instructions

If there are any of the following in this PR, provide proper instructions on how to upgrade:


Props relying on constants in `core/common` have been removed or replaced:
- `Card` `density` removed
- `Image` `shape` replaced with `rounded` and `circle` boolean props
- `Swatch` `shape` replaced with `rounded` and `circle` boolean props

Added `colorFocus` and `colorActive` props to the `A` component.

## Testing Instructions

- Use docz pages and/or testbed app to confirm components work as expected

Closes #196 
Closes #197 
Closes #149 
Closes #156 
Closes #151
